ext/mysqlnd/mysqlnd_wireprotocol.h

```diff
@@ -34,7 +34,7 @@ PHPAPI extern const char mysqlnd_read_body_name[];
 #define PACKET_FREE(packet) \
 	do { \
 		DBG_INF_FMT("PACKET_FREE(%p)", packet); \
-		if ((packet)->header.m->free_mem) { \
+		if ((packet)->header.m && (packet)->header.m->free_mem) { \
```

Comment:
This is not needed. It is already guaranteed.

Reply:
Removed